spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -166,27 +166,6 @@ def expect_retryable_error(error_class)
           end
         end
       end
-
-      context 'with PACT-related disability' do
-        let(:submission) do
-          create(:form526_submission,
-                 :with_pact_related_disabilities,
-                 user_uuid: user.uuid,
-                 auth_headers_json: auth_headers.to_json,
-                 saved_claim_id: saved_claim.id)
-        end
-
-        it 'sends an email' do
```

Comment:
This test assumed that `rrd_new_pact_related_disability` is enabled, and is no longer needed.